src/backend/common/queries/team_query.py

```diff
@@ -98,6 +99,18 @@ def _query_async(
             DistrictTeam.district_key == ndb.Key(District, district_key)
         ).fetch_async()
         team_keys = map(lambda district_team: district_team.team, district_teams)
+
+        matches = re.match(r"\d{4,}", district_key)
+        if matches is not None:
```

Comment:
I don't think we ever expect this to not match, so it might be cleaner just to throw if this parse fails instead of handling the `None` case.

Reply:
I'm trying to see if we validate the district key anywhere (and throw a 404) or if we expect to return an empty list for invalid district keys.

Reply:
Oh jk this is the query not the API 🤦 

yeah then this should be robust to incorrect keys and not throw an exception. ignore everything above 😛